cypress/e2e/gui/basic.cy.js

```diff
@@ -0,0 +1,96 @@
+describe('Basic', () => {
+  beforeEach(() => {
+    Cypress.on('window:before:load', window => {
+      window.document.cookie = 'cookieConsent=accepted'
+    })
+    cy.visit('/')
+  })
+
+  it('keeps the filters when coming back from the customer details view', () => {
+    cy.get('[data-testid="size-filter"]').select('Small')
+    cy.get('[data-testid="industry-filter"]').select('Finance')
+    cy.contains('button', 'View').click()
+    cy.contains('h2', 'Customer Details').should('be.visible')
+    cy.contains('button', 'Back').click()
```

Comment:
```suggestion
    cy.contains('button', 'Back').click()

```

E também deixaria uma linha em branco aqui, para separar a ação (_Act_ do padrão 3A) do _Assert_ (o terceiro A).

AAA - _Arrange_, _Act_, _Assert_ - pré-condição, ação e resultado esperado.

Reply:
O mesmo vale para outros testes onde pré-condição, ação e resultado esperado estão todos "grudados".